zerver/tests/test_signup.py

```diff
@@ -2425,15 +2426,9 @@ def generate_multiuse_invite_link(
 
         if date_sent is None:
             date_sent = timezone_now()
-        key = generate_key()
-        Confirmation.objects.create(
-            content_object=invite,
-            date_sent=date_sent,
-            confirmation_key=key,
-            type=Confirmation.MULTIUSE_INVITE,
-        )
-
-        return confirmation_url(key, self.realm, Confirmation.MULTIUSE_INVITE)
+        validity_in_days = 2
+        with patch("confirmation.models.timezone_now", return_value=date_sent):
+            return create_confirmation_link(invite, Confirmation.MULTIUSE_INVITE, validity_in_days)
```

Comment:
I think rather than this calling convention, we should just do:
```
        with patch("confirmation.models.timezone_now", return_value=date_sent):
            return create_confirmation_link(invite, Confirmation.MULTIUSE_INVITE, validity_in_days=2)
```

and make the parameter a kwarg-only parameter.